checkers/Handler.go

```diff
@@ -0,0 +1,60 @@
+// Handler provide routing for Top-level checker
+package checkers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/buaazp/fasthttprouter"
+	"github.com/go-kit/kit/log"
+	"github.com/valyala/fasthttp"
+	"os"
+)
+
+var (
+	handlerChecker Checker
+)
+
+type Handler struct {
+	Client fasthttp.Client
+	addr   string
+}
```

Comment:
health check is not performance critical functionality
try to avoid additional transitive dependencies